microapps/FeatureFormsApp/app/src/main/java/com/arcgismaps/toolkit/featureformsapp/screens/search/SearchViewModel.kt

```diff
@@ -96,6 +96,8 @@ class SearchViewModel @Inject constructor(
             types = listOf(PortalItemType.WebMap),
             searchQuery = "title:$searchQuery",
         ).apply {
+            // TO-DO: make this configurable
```

Comment:
is this comment intentional, if yes maybe use standard notation so it is noticible as a TODO
```suggestion
            // TODO: make this configurable
```

Reply:
Yeah, it was intentional. Updated!